tutorials/building-ai-application-function-calling/index.mdx

```diff
@@ -0,0 +1,272 @@
+---
+meta:
+  title: Agentic AI with function calling on open-weight LLMs
+  description: Learn how to implement function calling in your applications using a practical flight schedule example.
+content:
+  h1: Get started with agentic AI - building a flight assistant with function calling on open-weight Llama 3.1
```

Comment:
Unusual to have different meta and content titles, is it intended?

Reply:
@tgenaitay seems to be a forgotten older title? I guess the h1 is the right one

Reply:
Indeed I have had several attempts at the title, didn't know they have to be consistent between meta and body!